app.js

```diff
@@ -1,163 +1,227 @@
 'use strict';
 
-var store1 = {
+var storeQ = {
```

Comment:
This and the other variable names that you have for each store are not descriptive at all. Why default to these variable names when you could be using the camel cased version of "firstAndPike", etc?